NF.csv

```diff
@@ -750,9 +752,10 @@ parentSpecimenID,"A unique identifier (non-PII) that represents the parent speci
+modelSystemName,"A group of presumed common ancestry with clear-cut physiological but usually not morphological distinctions such as an animal model or cell line. EXAMPLE(S):  HEK293 (cell line), Minnesota5 (swine strain), DXL (poultry strain), RB51 (vaccine strain of Brucella abortus)","C57BL/6J,NOD scid gamma,B6;129S2-Trp53tm1Tyj Nf1tm1Tyj/J,B6;129-Trp53tm1Tyj Nf1tm1Tyj Suz12Gt(Betageo)1Khe/KcichJ,B6.129S2-Nf1tm1Tyj/J,B6.129S1-Nf1tm1Cbr/J,B6.129(Cg)-Nf1tm1Par/J,hiPSC-SCP,hiPSC,JHU 2-103-CL,JHU 2-002-CL,JHU 2-079-CL,JHU 2-103-PDX,JHU 2-002-PDX,JHU 2-079-PDX,human foreskin fibroblasts,hTERT NF1 ipNF05.5,hTERT NF1 ipNF05.5 (Mixed clones),hTERT NF1 ipn06.2 A,hTERT NF1 ipNF95.11b C/T,hTERT NF1 ipNF95.6,hTERT SC ipn02.8,hTERT NF1 ipnNF95.11c,hTERT NF1 ipNF95.11b C,hTERT NF1 ipnNF09.4,hTERT NF1 ipNF04.4,hTERT NF1 sipnNF95.12B,hTERT NF1 ipNF00.6,hTERT NF1 ipNF03.3,hTERT NF1 ipn02.3 2λ,hTERT SC ipn02.8,SZ-NF1,SZ-NF2,SZ-NF4,NF1,Lis42_NF1_1N,Lis47_NF1_2N,ST88-14,sNF96.2,sNF02.2,sNF94.3,GM11602,GM11601,KCL024,KCL025,5PNF_TdiPSsv_PM_6,5PNF_TdiPSsv_MM_4,7PNF_SiPSrv_PM_12,6PNF_SiPSrv_PM_2,3PNF_SiPSsv_MM_11,3PNF_FiPSsv_PM_2,NCC-MPNST5-C1,NCC-MPNST3-C1,NCC-MPNST4-C1,NCC-MPNST3-X2-C1,NCC-MPNST2-C1,STS-26T,YST-1,90-8,S520,S462,S462.TY,sc93.1,Fb93.1, JH-2-002-CL, JH-2-079-CL, JH-2-103-CL, icNF00.10a, cNF00.10a, icNF97.2a, cNF97.2a, icNF97.2b, cNF97.2b, icNF98.4d, cNF98.4d, i28cNF, 28cNF, icNF04.9a, cNF04.9a, icNF98.4c, cNF98.4c, Nf2-/- Schwann SC (mouse) (PMID26554010), SC4 [Mouse schwannoma], AC007-hTERT, NF2-/- AC007-hTERT, Ben-Men-1,M3 MPNST,NRG,HEK293",,FALSE,http://purl.obolibrary.org/obo/NCIT_C94171,neuro,,,list like,modelSystemName,Property,Biosample,annotationProperty,annotationProperty,Model_System,,
 nf2Genotype,"Genotype of NF2 gene in the biospecimen from which the data were derived, if known","-/-, +/-, +/+, +/?, -/?, Unknown",,FALSE,,neurofibromatosis,,,,nf2Genotype,Property,Biosample,annotationProperty,,Genotype,,
 nf1Genotype,"Genotype of NF1 gene in the biospecimen from which the data were derived, if known","-/-, +/-, +/+, +/?, -/?, Unknown",,FALSE,,neurofibromatosis,,,,nf1Genotype,Property,Biosample,annotationProperty,,Genotype,,
+nf1Variant,"Variant (mutation) in the NF1 gene/protein in the biospecimen from which the data were derived, if known/applicable.","R816X,E178X,L578P,R1276Q,R1513X,V1621R,R2237X,Y2264X6",,FALSE,,neurofibromatosis,,,,nf1Mutation,Property,Biosample,annotationProperty,,Genotype,,"See https://www.hgmd.cf.ac.uk/docs/mut_nom.html#general for nomenclature."
```

Comment:
```suggestion
nf1Variant,"Variant (mutation) in the NF1 gene/protein in the biospecimen from which the data were derived, if known/applicable.","R816X,E178X,L578P,R1276Q,R1513X,V1621R,R2237X,Y2264X6",,FALSE,,neurofibromatosis,,,,nf1Mutation,Property,Biosample,annotationProperty,,Genotype,,"See https://www.hgmd.cf.ac.uk/docs/mut_nom.html#general for nomenclature."
```

 I wonder if instead of tracking the variants here, we could reference the variant UUIDs from the NF Tools Database? What do you think? https://www.synapse.org/#!Synapse:syn26486835/tables/query/eyJzcWwiOiJTRUxFQ1QgKiBGUk9NIHN5bjI2NDg2ODM1IHdoZXJlIHByb3RlaW5WYXJpYXRpb24gPSAncC5Bcmc4MTZUZXInIiwgImFkZGl0aW9uYWxGaWx0ZXJzIjpbXSwgInNlbGVjdGVkRmFjZXRzIjpbXSwgImluY2x1ZGVFbnRpdHlFdGFnIjp0cnVlLCAib2Zmc2V0IjowLCAibGltaXQiOjI1LCAic29ydCI6W3siY29sdW1uIjoicHJvdGVpblZhcmlhdGlvbiIsICJkaXJlY3Rpb24iOiJERVNDIn1dfQ== (in this case, I believe we'd use the "transgenic" one)

Reply:
Tbh, I think using UUIDs is going to be confusing and error-prone for contributors and not helpful if this is ever surfaced in the portal. We just don't have an interface where the label can be selected and the actual UUID is used for the index. Besides, there are only 3 variants tracked in the NF Tools Database currently. Moreover, the nomenclature is standardized enough that it will be easier to link than most things.